src/app/body/product/product.component.scss

```diff
@@ -7,17 +7,27 @@
     display: flex;
     flex-direction: column;
     margin-right: 32px;
-    min-width: 48%;
+    min-width: 50%;
+    max-width: 50%;
     margin-top: 30px;
+    img {
+      width: 100%;
+      height: 100%;
+      border-radius: 12px;
+      object-fit: cover;
+    }
     &-top {
-      img {
-        width: 100%;
-      }
+      width: 100%;
+      max-height: 415px;
     }
     &-bottom {
       margin-top: 26px;
       display: flex;
       justify-content: space-between;
+      div {
+        width: calc(50% - 20px);
+        height: 218px;
+      }
```

Comment:
it is a bad selector, try to find more sematic name for it

Reply:
fixed